### PR TITLE
It is back.

### PR DESCRIPTION
**Pull request by Nebyt**
Added back e2e tests. No Cucumber now.

### CODE REVIEW

**Comment by neuodev** [comment]
[autoscirpt] Jira Ticket: **[Move e2e-tests to the yoroi-extension repository](https://emurgo.atlassian.net/browse/yoext-1076)**

[](__JIRA_COMMENT_FLAG)

**Review by Nebyt — COMMENTED** [review]
/regression-check

**Review by Nebyt — COMMENTED** [review]
/regression-check

**Review by Nebyt — COMMENTED** [review]
/regression-check

**Comment by zuzunker** [comment]
> It is back

image

**Review by Nebyt — COMMENTED** [review]
/regression-check

**Review by Nebyt — COMMENTED** [review]
/regression-check

**Review by Nebyt — COMMENTED** [review]
/regression-check

**Review by Nebyt — COMMENTED** [review]
/regression-check

**Review by Nebyt — COMMENTED** [review]
/regression-check

**Review by Nebyt — COMMENTED** [review]
/regression-check

**Review by Nebyt — COMMENTED** [review]
/regression-check

**Review by Nebyt — COMMENTED** [review]
/regression-check

**Review by Nebyt — COMMENTED** [review]
/regression-check

**Review by Nebyt — COMMENTED** [review]
/regression-check

**Review by Nebyt — COMMENTED** [review]
/regression-check

**Review by Nebyt — COMMENTED** [review]
/regression-check

**Review by Nebyt — COMMENTED** [review]
/regression-check

**Review by Nebyt — COMMENTED** [review]
/regression-check

**Review by Nebyt — COMMENTED** [review]
/dapp-check

**Review by Nebyt — COMMENTED** [review]
/dapp-check

**Review by Nebyt — COMMENTED** [review]
/check

File: install-all.sh [hunk 1]
```diff
@@ -1,3 +1,5 @@
 npm i --legacy-peer-deps \
-&& npm i --legacy-peer-deps --prefix packages/yoroi-extension
+&& npm i --legacy-peer-deps --prefix packages/yoroi-extension \
+&& npm i --prefix packages/e2e-tests chromedriver --chromedriver_version=LATEST \
+&& npm i --prefix packages/e2e-tests
```
Comment (zuzunker) [inline install-all.sh]: @Nebyt , why this needs to happen twice?

Comment (Nebyt) [reply install-all.sh]: My mistake, the line 4 should be deleted.

Comment (Nebyt) [reply install-all.sh]: Oh, no, wait. This change is specifically for the chromedriver to install the latest one.

**Review by Nebyt — COMMENTED** [review]
/check

**Review by zuzunker — DISMISSED** [review]

**Review by zuzunker — APPROVED** [review]

**Pull request merged**